.github/ISSUE_TEMPLATE/02-docs.yml

```
@@ -0,0 +1,46 @@
+name: Documentation Bug
+labels: ["status/needs-triage", "type/documentation"]
+description: Report an issue with Bottlerocket's documentation.
+body:
+  - type: markdown
+    attributes:
+      value: |
+        This bug report form is for **docuemntation issues** related to Bottlerocket.
+
+        A few tips before submitting a new issue:
+        - Search for similar issues, including [closed issues](https://github.com/bottlerocket-os/bottlerocket/issues?q=is%3Aissue+is%3Aclosed),
+        - Check if this has been reported on the [project website repo](https://github.com/bottlerocket-os/project-website).
+
+  - type: dropdown
+    id: doctype
+    attributes:
+      label: I noticed this documentation problem when I was...
+      options:
+        - Using Bottlerocket
+        - Developing Bottlerocket
```

What do you think about more fine-grained "using" options?

```suggestion
        - Running Bottlerocket
        - Learning Bottlerocket
        - Developing Bottlerocket
```

---

Not sure about 'learning' but I'm open. Someone learning could be either hacking away on the internals _or_ someone trying to understand how to setup bottlerocket on a server (more similar to 'running'). So I'm not sure what that clarifies.